examples/c++-ros2-dataflow/run.rs

```diff
@@ -0,0 +1,173 @@
+use eyre::{bail, Context};
+use std::{env::consts::EXE_SUFFIX, path::Path};
+use tracing::metadata::LevelFilter;
+use tracing_subscriber::Layer;
+
+#[derive(Debug, Clone, clap::Parser)]
+pub struct Args {
+    #[clap(long)]
+    pub run_dora_runtime: bool,
+}
+
+#[tokio::main]
+async fn main() -> eyre::Result<()> {
+    let Args { run_dora_runtime } = clap::Parser::parse();
+
+    if run_dora_runtime {
+        return tokio::task::block_in_place(dora_daemon::run_dora_runtime);
```

Comment:
I think that you're not using the latest version of dora main branch, as the runtime has now been moved to the cli.

You can take the inspiration from https://github.com/dora-rs/dora/pull/410/files#diff-f7eaba92236631b04e38329e55de2c118e1950d18d922a8b922d839c57217750 where I have simplified the run script to make it also more readable

Reply:
Ah yeah, looks like I forgot to update this when rebasing this branch. Should be fixed in 57de6973.

I also pushed da44f75 to run the new example on CI so that we catch such errors automatically.